xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/Logic.js

```diff
@@ -25,7 +25,7 @@ define('xwiki-livedata', [
   "xwiki-livedata-source",
   "xwiki-json-merge",
   "xwiki-livedata-polyfills"
-], function (
+], function(
```

Comment:
sorry for the bad large diff, I'll fix that. The only interesting change is the introduction of the getEditEntryProperty method at the bottom of the file.